SyntheticsRunTestsTask/__tests__/fixtures.ts

```diff
@@ -37,10 +37,10 @@ const runMockedTask = (mockName: string): MockTestRunner => {
   // See `16.15.0` in `.node-version`
   const nodeVersion = 16
 
-  const task = new MockTestRunner(file)
-  task.run(nodeVersion)
+  const task = await new MockTestRunner().LoadAsync(file)
+  await task.runAsync(nodeVersion)
```

Comment:
This now had to be async, otherwise [this line](https://github.com/microsoft/azure-pipelines-task-lib/blob/c1183894df5189eb7479b901633f5ccf460c96a5/node/mock-test.ts#L200-L201) does an infinite loop for some reason.